src/chart-review/chart-review.component.tsx

```diff
@@ -123,7 +77,8 @@ export default function ChartReview(props: any) {
                   >
                     <Link
                       to={
-                        defaultPath + item.path + (tabHistory[item.path] || "")
+                        tabHistory[item.path.substr(1)] ||
```

Comment:
why the index `1` specifically? what does the item at index `1` signify?

Reply:
Because the paths have a "/" in front, e.g. "/overview". See https://github.com/openmrs/openmrs-esm-patient-chart/blob/tabbed-view/src/view-components/core-views.tsx#L25 for an example config.

The tabHistory is using the  path without the "/" to store the last route.